### ISSUE TITLE
--drop-row-if-null=col1

### ISSUE THREAD

**Issue by seamusabshere** [faradayio/scrubcsv#9]
here's a good list of things to consider NULL, courtesy of https://bigml.com/api/sources

```
["", "N/A", "n/a", "NULL",
"null", "-", "#DIV/0", "#REF!",
"#NAME?", "NIL", "nil", "NA",
"na", "#VALUE!", "#NULL!",
"NaN", "#N/A", "#NUM!", "?"]
```

**Comment by seamusabshere** [comment]
probably missing mysql's `\N`

**Comment by emk** [comment]
Implemented in v0.1.6.

Note that the NULL columns are normalized using `--null REGEX`, and you can specify columns that must not be NULL using `--drop-row-if-null COL`, which you can pass more than once. Thank you for the suggestion!
